docs/scripts/buildApi.ts

```diff
@@ -476,11 +524,14 @@ Page.getInitialProps = () => {
   console.log('Built markdown docs for', reactAPI.name);
 
   await annotateComponentDefinition({ api: reactAPI, component: componentObject });
-  await annotateClassesDefinition({
-    api: reactAPI,
-    component: componentObject,
-    prettierConfigPath,
-  });
+
+  if (!nonJSSComponent) {
```

Comment:
For the components generated with `withStyled` the info and descriptions are gathered from the component's static styles field. However, for the unstyled & the components generated with styled() this info does not exists, but is taken from the ts definition of the classes prop. That is why we do not need to update the annotation for the classes definition for these components.